spec/requests/posts_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get posts_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get posts_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get posts_path
+      expect(response.body).to include('Here is a list of all posts')
+    end
+  end
+
+  describe 'GET #show' do
+    let(:post) { Post.create(id: 1, title: 'Sample Post', text: 'This is the content of the post') }
+
+    it 'returns a successful response' do
+      get post_path(post.id)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get post_path(post)
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get post_path(post)
+      expect(response.body).to include('Post Details')
+    end
+  end
+end
```

Comment:
- [ ] Hi @Simpleshaikh1, all your requests(controller) tests are failing as you can see on the screenshot below: I also realized that you deleted the users_spec.rb file. Remember in the requirements of this project, it is pointed out that `users_spec.rb` file should be in `spec/requests` directory and you should tests both `index` and `show` actions for `Users` controller as you have done with the `Posts` controller. Please let's kindly fix :wrench: this issue.
![Screenshot from 2023-10-31 19-17-22](https://github.com/Simpleshaikh1/blog-app/assets/63087491/c3c675c2-47bb-4626-8ec9-3ae464dce2b2)